<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNetworkSet</class>
 <widget class="QDialog" name="DialogNetworkSet">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>前端服务器</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrontServerIp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrontServerPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectFrontServer">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接前端服务器</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>791</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>本地接收实时数据的代理服务器</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLocalAgentReceiverPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartLocalAgentReceiver">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动本地代理服务</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLocalAgent">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>791</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>数据中心</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataCenterIp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataCenterPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectDataCenter">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接数据中心</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>791</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>A12系统</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA12Ip">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>作业队：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA12TeamName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA12TeamPwd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectA12">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接A12系统</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>331</width>
     <height>571</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
